### ISSUE TITLE
names are automatically matched correctly, yet nothing else is

### ISSUE THREAD

**Issue by Redhawk18** [Dingmatt/AMSA#39]
every show has the [anidb-(id)] in the title, so clearly something is being changed, but none of the other metadata or images are downloading/updating.

**Comment by Redhawk18** [comment]
does the plex user have to own these files?

**Comment by Dingmatt** [comment]
Plex will modify them periodically, so the user it runs under will need read/write permissions

**Comment by Redhawk18** [comment]
How do i dis-allow anidb from using japanese names?

**Comment by Redhawk18** [comment]
everything works perfectly except that

**Comment by Dingmatt** [comment]
Go to the agent's settings, there you can tweak the preferred language settings. Though by default the agent should be defaulting to English where available.

![image](https://user-images.githubusercontent.com/14905596/204380442-a8eb4655-b9ed-4ab0-b97b-8f305dd6b3b3.png)

**Comment by kikoptr** [comment]
I'm sorry to reopen this ticket but I have the same problem and I can't solve it.
I have checked the Cache files and in my case these are generated, one of them without XML extension so I do not know if it is a bug or a change in the latest version:
```
anime-list-corrections.xml
anime-list-master.xml
anime-movieset-list.xml
anime-titles
```
Other than that, in the Plex log when I'm trying to download metadata or apply an ID correction I've gotten this a couple of times:
```
tvdb:71: FutureWarning: The behavior of this method will change in future versions. Use specific 'len(elem)' or 'elem is not None' test instead.
The '--scan' operation is deprecated and will be removed in future versions of Plex Media Server.
```
Once I have identified the series, the titles remain with the "anidb" and no posters or descriptions are downloaded. What can I do to fix this?

Thanks.
